sections/webappapis.include

```diff
@@ -4709,5 +4710,84 @@
     </pre>
   </div>
 
+<h3 id="animation-frames">Animation Frames</h3>
+
+  <p>Each <code>Document</code> associated with a <a>top-level browsing context</a>
```

Comment:
Let's use Bikeshed's markdown minimal syntax for new content:
* No need for `<p>`, just separate paragraphs by a newline
* Lists use numbers (`1., 2., etc.`) with proper intentation as seen above (line 982 for example).

Additionally,

1. Let's drop the 'algorithm' attribute below. Seems like a nice idea, but has no bearing on our spec processing
2. `<code>Document</code>` and `<code>Window</code>` should be `{{Document}}` and `{{Window}}` respectively, so that they link-up.